foundation-extensions/mobile-core/platform-sercvies/README.md

```diff
@@ -1,6 +1,46 @@
 # Platform Services
 
-Currently, the AEP SDK only provides the capability for overriding the Adobe-provided network stack \(for all Adobe extensions\).
+The Platform Services are provided by the Adobe Experience Platform Mobile SDKs as part of the Mobile Core extension. These services provide shared functionality throughout the SDK that can be shared by extensions. For example, services provide shared functionality for networking, data queuing, caching, and more.
+
+{% hint style="info" %}
+This feature is only available in [Android Core 1.8.0](https://aep-sdks.gitbook.io/docs/release-notes#android-core-1-8-0) or later and [iOS Core 3.0.0](https://aep-sdks.gitbook.io/docs/release-notes#adobe-experience-platform-ios-core-sdks) or later.
+{% endhint %}
+
+| Service  | Android SDK | iOS SDK |
+| :----------------------------------------------------------- | :----------------------------------------------------------- | :---------------------------------------------------------- |
+| [Networking](./network-servcie.md) | Core 1.8.0 + | AEPCore 3.0.0 + |
+| [DataQueue](./data-queue-service.md) | Core 1.8.0 + | AEPCore 3.0.0 + |
+| [SystemInfoService/DeviceInforming](./system-device-info-service.md) | Core 1.8.0 + | AEPCore 3.0.0 + |
+
+## Accessing Services
+
+The MobileCore extension provides a shared `ServiceProvider`, responsible for accessing the current set of provided services.
+
+The following code snippet shows how to access `Network Service`  as an example.
+
+{% tabs %}
+{% tab title="Android" %}
+
+```java
+import com.adobe.marketing.mobile.services.ServiceProvider;
+
+// Get an instance of the current network service
+Networking networkService = ServiceProvider.getInstance().getNetworkService();
+```
+
+{% endtab %}
+
+{% tab title="iOS(AEP 3.x)" %}
```

Comment:
Add a space in between `iOS (AEP 3.x)`. Same comment for all occurences.

Reply:
There are few more of these ☝️

Reply:
It's fixed in the PR, but not shown correctly on this page, you can find the updated doc on the "Files Changed" page.